crates/sequencer/src/sequencer.rs

```diff
@@ -532,6 +532,13 @@ where
                     });
                 }
 
+                info!(
+                    "New block #{}, DA #{}, Tx count: #{}",
+                    l2_height,
+                    da_block.header().height(),
+                    txs_count,
```

Comment:
```suggestion
                    txs_count = txs.len(),
```

Reply:
You would have to clone `txs` because it moved in the lines before which is more expensive than just getting the count before the move.

Reply:
Oh, right!